mrmustard/math/backend_manager.py

```diff
@@ -864,6 +876,19 @@ def isnan(self, array: Tensor) -> Tensor:
         """
         return self._apply("isnan", (array,))
 
+    def issubdtype(self, arg1, arg2) -> bool:
+        r"""
+        Whether the ``arg1`` is a typecode lower/equal in type hierarchy to ``arg2``.
+
+        Args:
+            arg1: The first argument.
+            arg2: The second argument.
```

Comment:
```suggestion
            arg1: The object to be tested
            arg2: The object to be compared against
```
or something similar? Maybe not so necessary, given the first line and what's in "Returns:", but doesn't seem to useful as written either.